### PR TITLE
Make the shadowJar multi-release.

### PR DESCRIPTION
**Pull request by gbggrant**
### Description

This PR changes the jar packaging to be multi-release.  This addresses the issues surfaced with the upgrade to gkl 0.8.8

----

### Checklist (never delete this)

Never delete this, it is our record that procedure was followed. If you find that for whatever reason one of the checklist points doesn't apply to your PR, you can leave it unchecked but please add an explanation below.

#### Content
- [ ] Added or modified tests to cover changes and any new functionality
- [ ] Edited the README / documentation (if applicable)
- [ ] All tests passing on Travis

#### Review
- [ ] Final thumbs-up from reviewer
- [ ] Rebase, squash and reword as applicable

For more detailed guidelines, see https://github.com/broadinstitute/picard/wiki/Guidelines-for-pull-requests

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -141,7 +141,9 @@ tasks.withType(Jar){
         attributes 'Main-Class': 'picard.cmdline.PicardCommandLine',
                 'Implementation-Title': 'Picard',
                 'Implementation-Vendor': 'Broad Institute',
-                'Implementation-Version': version
+                'htsjdk-Version': htsjdkVersion,
```
Comment (lbergelson) [inline build.gradle:144]: This is unnecessary for the fix but seems like a fine addition...

**Review by lbergelson — APPROVED** [review]
Looks good

File: build.gradle (hunk 1 above)
Comment (gbggrant) [reply build.gradle:144]: Yes.  I was inspired by your example.

**Comment by lbergelson** [comment]
We could add the following test to the script section of build.gradle.  It doesn't add very much time to the build.
```
 # build the docs and shadowJars
 - ./gradlew
 # run a non-trivial tool to be sure it doesn't explode
 - java -jar build/libs/picard.jar MarkDuplicates -I testdata/picard/sam/aligned_queryname_sorted.bam -O out.bam --METRICS_FILE out.metrics

```

**Comment by lbergelson** [comment]
@gbggrant I think we have to merge my other pr that fixes that doc problem on java 11 before running the complete `./gradle` build is going to pass.

**Comment by lbergelson** [comment]
Sorry, try replacing `./gradlew` with `./gradlew currentJar`

**Pull request merged**